modin/test/storage_formats/pandas/test_internals.py

```diff
@@ -60,6 +62,57 @@ def put(x):
     virtual_row_partition_class = PandasOnDaskDataframeRowPartition
 
 
+def construct_modin_df_by_scheme(pandas_df, partitioning_scheme):
+    """
+    Build ``modin.pandas.DataFrame`` from ``pandas.DataFrame`` according the `partitioning_scheme`.
+
+    Parameters
+    ----------
+    pandas_df : pandas.DataFrame
+    partitioning_scheme : dict[{"row_lengths", "column_widths"}] -> list of ints
+
+    Returns
+    -------
+    modin.pandas.DataFrame
+    """
+    row_partitions = split_result_of_axis_func_pandas(
+        axis=0,
+        num_splits=len(partitioning_scheme["row_lengths"]),
+        result=pandas_df,
+        length_list=partitioning_scheme["row_lengths"],
+    )
+    partitions = [
+        split_result_of_axis_func_pandas(
+            axis=1,
+            num_splits=len(partitioning_scheme["column_widths"]),
+            result=row_part,
+            length_list=partitioning_scheme["column_widths"],
+        )
+        for row_part in row_partitions
+    ]
+
+    md_df = from_partitions(
```

Comment:
I really don't like the name `md_df` for a modin df var. That makes our main point to import modin as `pd` kind of vague. So I prefer to use `modin_df` as we do in most places of our tests.